src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/sentencing/SentencingService.kt

```diff
@@ -37,4 +37,12 @@ class SentencingService(@Qualifier("sentencingApiWebClient") private val webClie
       .retrieve()
       .awaitBodyOrNullWhenNotFound<Unit>()
   }
+
```

Comment:
new DPS endpoint to just set currentTerm - but for ease they are using the existing DTO - all fields will be ignored